packages/cookies/src/response-cookies.ts

```diff
@@ -102,3 +108,82 @@ function normalizeCookie(cookie: ResponseCookie = { name: '', value: '' }) {
 
   return cookie
 }
+
+/**
+ * @source https://github.com/nfriedly/set-cookie-parser/blob/master/lib/set-cookie.js
+ *
+ * Set-Cookie header field-values are sometimes comma joined in one string. This splits them without choking on commas
+ * that are within a single set-cookie field-value, such as in the Expires portion.
+ * This is uncommon, but explicitly allowed - see https://tools.ietf.org/html/rfc2616#section-4.2
+ * Node.js does this for every header *except* set-cookie - see https://github.com/nodejs/node/blob/d5e363b77ebaf1caf67cd7528224b651c86815c1/lib/_http_incoming.js#L128
+ * React Native's fetch does this for *every* header, including set-cookie.
+ *
+ * Based on: https://github.com/google/j2objc/commit/16820fdbc8f76ca0c33472810ce0cb03d20efe25
+ * Credits to: https://github.com/tomball for original and https://github.com/chrusart for JavaScript implementation
+ */
+function splitCookiesString(cookiesString: string) {
```

Comment:
**suggestion:** we already have this function in `packages/node-utils/src/edge-to-node/headers`. Can we share it between the two packages? maybe a third package called `internals-utils`? @Kikobeats, @javivelasco, what do you think?

Reply:
yes, we can create a package for that, also for testing it properly.